doc/ref/run.xml

```diff
@@ -391,9 +399,10 @@ redirect the output to a printer, you may want to increase this value.
 <Item>
 Further arguments are taken as filenames of files that are read by &GAP;
 during startup, after the system  and private init   files are read,  but
-before the first prompt is  printed.
-The files are read in the order in which they appear on the command line.
-&GAP; only accepts up to 14 filenames on the command line.
+before the first prompt is  printed. Filenames ending with <C>.tst</C>
+are processed by <Ref Func="Test" />, all other files by <Ref Oper="Read" />.
+These files and also commands specified via the <C>-c</C> option are processed
+in the order in which they appear on the command line.
 If a file cannot be opened &GAP; will print an error message and will abort.
```

Comment:
Off-topic, but: this final sentence (which I did not touch, it was here before) is factually wrong: if the file cannot be opened then GAP prints `Could not read file "FILANEM".` and happily chugs on. Should I update the documentation to match the code, or the other way around? Personally I'd prefer to change the code, as it almost always means I made a typo, and sometimes I don't notice; if GAP would instead exit with a non-zero exit code, that'd help me find bugs earlier. But on the downside, who knows, maybe this change would break someones workflow somewhere?